ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/MSVS_2010/tasks/main.yml

```diff
@@ -20,6 +20,10 @@
   when: (vs2010_installed.stat.exists == false)
   tags: MSVS_2010
 
+- name: Register Visual Studio 2010 DIA SDK shared libraries
+  raw: 'for /r "C:\Program Files (x86)\Microsoft Visual Studio 10.0\DIA SDK" %F in ( msdia*.dll ) do regsvr32 /s "%~F"'
```

Comment:
What does the wildcard pick up?  Isn't it sufficient to register `msdia100.dll` alone?

Reply:
Without the wildcard, it will invoke regsvr32 in *every* subdirectory, not just those with the DLLs we seek to register.

Reply:
Ok, so I was imprecise :-), I meant why the need for the loop over the wildcard match, i.e. why not just
`regsvr32 /s "c:\Program Files (x86)\Microsoft Visual Studio 10.0\DIA SDK\bin\amd64\msdia100.dll"`

Reply:
There are two DLLs and I think both should be registered. If you prefer, we could say:
```
regsvr32 /s "c:\Program Files (x86)\Microsoft Visual Studio 10.0\DIA SDK\bin\msdia100.dll"
regsvr32 /s "c:\Program Files (x86)\Microsoft Visual Studio 10.0\DIA SDK\bin\amd64\msdia100.dll"
```
but I wasn't sure whether/how you would express two commands in ansible.